manuscript.md

```diff
@@ -4,13 +4,15 @@ Yi Wang, ..., Diego Barrios Romero , ..., Li Jin*
 
 Yi Wang: Ministry of Education Key Laboratory of Contemporary Anthropology, Collaborative Innovation Center for Genetics and Development, School of Life Sciences, Shanghai, China. Human Phenome Institute, Fudan University, Shanghai, China.
 
-Diego Barrios Romero: Germany  **TODO**: enter your full address
+Diego Barrios Romero: Fraunhofer Institute for Digital Medicine MEVIS, Bremen, Germany.
 
 Li Jin: Human Phenome Institute, Fudan University, Shanghai, China. State Key Laboratory of Genetic Engineering, Collaborative Innovation Center for Genetics and Development, School of Life Sciences, Fudan University, Shanghai, China.
 
 ----------------------------------------
 **Introduction**
 
+wyhash was born out of the desire to find a 64-bit hash function and PRNG that are solid, portable, fastest and simplest.
```

Comment:
Suggest "wyhash was born out of the desire to find a 64-bit hash function and PRNG that are solid, portable, fast and simple."

BTW, hope you don't mind me providing a few spelling/grammar suggestions (I'm a native English speaker) 😄